src/placeholder/empty.tsx

```diff
@@ -0,0 +1,187 @@
+import { sva } from "../../styled-system/css";
+import { Box } from "../../styled-system/jsx";
+import { Icon } from "../common/Icon";
+import { Heading } from "../typography/Heading";
+import { Paragraph } from "../typography/Paragraph";
+
+import type * as React from "react";
```

Comment:
Why is this imported with *? Does this cause a problem with tree-shaking?

Reply:
I've changed to import the single type we've. I choose this way if we've a lot of imports.